packages/xdl/src/detach/StandaloneContext.ts

```diff
@@ -137,6 +137,10 @@ export class StandaloneContextService extends StandaloneContext {
   ) {
     super();
   }
+
+  isAnonymous = () => {
+    return true;
+  };
```

Comment:
Yes, it's safe to assume – this function is used only in one place in `IosWorkspace` file. Also this function can be removed after release because all iOS contexts become anonymous as the user context will no longer be a case.

Imho this `isAnonymous` function in `StandaloneContext` shouldn't have such implementation. Isn't class inheritance intended for such purpose? Using `instanceof` here doesn't make much sense – it should just return `false` here and be overridden in the subclass (`StandaloneContextService`) which is actually what I did.

Reply:
Sounds like a very good solution and insight—why not fix `StandaloneContext`'s implementation, while you're at it? 🙏